lib/components/SCardHeader.vue

```diff
@@ -54,8 +76,22 @@ export default defineComponent({
       `round-${props.round}`
     ])
 
+    function getIcon(icon: any, disabled: boolean) {
```

Comment:
Let's import `ActionIconType` from `Card` and use it here 🙌 

```suggestion
    function getIcon(icon: ActionIconType | object, disabled: boolean) {
```